.github/workflows/go.yml

```diff
@@ -0,0 +1,50 @@
+name: Go
```

Comment:
@vmihailenco I'm developing on a Windows machine and this was the quickest way to run the tests. 

Do you want me to remove this github action file?

Reply:
I don't mind to try Github workflow. Do I have to do anything to enable it?